src/us/kbase/workspace/test/kbase/KBaseWorkspaceConfigTest.java

```diff
@@ -521,8 +517,8 @@ public void configMaximalS3AndDelegateTypes() throws Exception {
 				"type-delegation-target=http://localhost:12345\n" +
 				"mongodb-retrywrites=true\n" +
 				"mongodb-user=muser\n" +
-				"auth-service-url=" + AUTH_LEGACY_URL + "\n" +
 				"auth2-service-url=" + CI_SERV + "auth\n" +
+				"dont-trust-x-ip-headers=true\n" +
```

Comment:
do you need a corresponding test with `"dont_trust_x_ip_headers=true\n"`? I couldn't see one below.

Reply:
I'm guessing you missed these lines maybe? https://github.com/kbase/workspace_deluxe/pull/618/files#diff-4b9e468fe59a05ef0feeaaa0f80dcfbbaf63767992679b8ae04a1be581cd9244R575-R609

Reply:
If you're talking about not having it printed in the param report, the legacy key is really old and has never been documented and I suspect is not used at all, so I didn't bother to put it in the report. I don't think this feature is used at all, period, actually, but it was less work to support it that to confirm that.

Reply:
I was talking about the printed version -- your explanation makes sense so 👍